notebooks/end-to-end-structured/labs/4c_keras_wide_and_deep_babyweight.ipynb

```diff
@@ -11,7 +11,7 @@
     "1. Set CSV Columns, label column, and column defaults\n",
```

Comment:
<p>* Add back the "Lab Task" or "Exercise" mention to help figuring out at a glance what should be done</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>